inf_personal_usuario infPersonal
string emocion
pos_usuario posicion